hack/patch.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -xe
+
+kubectl=./cluster-up/kubectl.sh
+
+function wait-for-daemonset() {
+    retries=10
+    while [[ $retries -ge 0 ]]; do
+        sleep 3
+        ready=$($kubectl -n $1 get daemonset $2 -o jsonpath="{.status.numberReady}")
+        required=$($kubectl -n $1 get daemonset $2 -o jsonpath="{.status.desiredNumberScheduled}")
+        if [[ $ready -eq $required ]]; then
+            #echo "Succeeded"
+            break
+        fi
+        ((retries--))
+    done
+}
+
+BAZEL=bazelisk PUSH_TARGETS="virt-controller virt-handler virt-launcher" ./hack/bazel-push-images.sh
+
+registry_port=$(./cluster-up/cli.sh ports registry)
+registry_url=localhost:$registry_port
+
+handler_sha256=$(skopeo inspect docker://$registry_url/kubevirt/virt-handler:latest --tls-verify=false | jq ".Digest" -r)
```

Comment:
If you call `source hack/parse-shasums.sh` you should have the following env variables available right away:

```
VIRT_API_SHA
VIRT_CONTROLLER_SHA
VIRT_HANDLER_SHA
VIRT_LAUNCHER_SHA
```

and don't need skopeo installed.

Reply:
nice, done.